js/hexaworld.js

```diff
@@ -0,0 +1,229 @@
+var hexaworld = {
+
+  $schema: "http://json-schema.org/schema#",
+  id: "https://github.com/hexaworld/hexaworld-schema/schema.js",
+
+  definitions: {
+
+    // basic definitions
+
+    pixelCoord: {
```

Comment:
I think that "position" of an object should always be its position in pixels relative to the "position" of the object that it is inside - and that the axial coordinate system or the sector coordinate system should be referred to differently